src/GurobiSolverInterface.jl

```diff
@@ -35,11 +35,11 @@ function copy(m::GurobiMathProgModel)
                                copy(m.obj), 
                                copy(m.lb), 
                                copy(m.ub), 
-                               deepcopy(lazycb),
-                               deepcopy(cutcb),
-                               deepcopy(heuristiccb), 
-                               deepcopy(infocb), 
-                               deepcopy(options))
+                               deepcopy(m.lazycb),
```

Comment:
It's a bit unclear if it's safe to copy callbacks. How would this work if the callbacks hold a reference to the old model, for example?